fix: Date Selector not working on Highlight Creation

aryanas159 opened the pull request:
…iting Dialogs

fixed issue #1756: Date Selector not working on Highlight Creation/Editing Dialogs


## Description
added pointer-events-auto to PopoverContent's class names.
<!-- 
Please do not leave this blank 
This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 
-->
fixes #1756 
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
<!-- 
Please use this format link issue numbers: Fixes #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->

## Mobile & Desktop Screenshots/Recording

https://github.com/open-sauced/app/assets/114330931/57aca967-d6e5-4803-8611-add6f1f6b1d7


https://github.com/open-sauced/app/assets/114330931/1149b9ff-3d0d-47e1-a376-65a2855fc6b5

s

<!-- Visual changes require screenshots -->


## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?



## [optional] What gif best describes this PR or how it makes you feel?



<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
  
  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.
  
  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this wil

https://github.com/open-sauced/app/assets/1143

https://github.com/open-sauced/app/assets/114330931/28ef2df4-f097-4735-916a-68c932ef1199


https://github.com/open-sauced/app/assets/114330931/a5513309-3b89-4eea-9e9d-5bc541810972

30931/6b46f6e9-437f-49d3-add9-34b17bba516a


https://github.com/open-sauced/app/assets/114330931/0f596880-b1b3-412f-8a2d-47920db65ff8

l be possible.
  - ✅ Provide tests for your changes.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

RitaDee commented on components/molecules/ContributorHighlight/contributor-highlight-card.tsx:
```diff
@@ -652,7 +652,7 @@ const ContributorHighlightCard = ({
                           </button>
                         </PopoverTrigger>
-                        <PopoverContent className="w-auto p-0 bg-white">
+                        <PopoverContent className="w-auto p-0 bg-white pointer-events-auto">
```
LGTM 🍕

babblebey commented:
Hey @aryanas159

Remove the `issue***:` from the PR title, leaving only the `fix: Bug******`.

This will let you satisfy PR compliance.

aryanas159 commented:
> Hey @aryanas159
> 
> Kindly remove the `issue 1756:` from the PR title, leaving only the `fix: Bug******`.
> 
> This will let you satisfy PR compliance.

Thanks for the heads up @babblebey 
I have changed the title.

aryanas159 commented:
> For ease of review next time @aryanas159, considering this PR fixes something very visual, you might want to include screencasts (i.e. picture or video) to demo the changes you made.
> 
> I would've loved to see a video demo for this one though, do you think you can get that!? 😉
> 
> Add it right under the `Mobile & Desktop Screenshots/Recordings` section on the PR Description template 😉

@babblebey  I though I had added the screencasts, but for some reason they weren't uploaded, I have added them again.

RitaDee commented:
> > For ease of review next time @aryanas159, considering this PR fixes something very visual, you might want to include screencasts (i.e. picture or video) to demo the changes you made.
> > I would've loved to see a video demo for this one though, do you think you can get that!? 😉
> > Add it right under the `Mobile & Desktop Screenshots/Recordings` section on the PR Description template 😉
> 
> @babblebey I thought I had added the screencasts, but for some reason they weren't uploaded, I have added them again.

I can see it on my end.

babblebey reviewed (commented):
🍕

OgDev-01 reviewed (approved):
LG\TM
